test/performance/broker-imc/continuous/100-broker-imc-setup.yaml

```diff
@@ -0,0 +1,182 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: imc
+  namespace: default
```

Comment:
@chizhg @slinkydeveloper well, it DIFFs in name/namespace.

But I agree, it's good to potentially reuse "common" setup yaml files...